### DISCUSSION TITLE
`cmake` will be added to the PATH so updates to the package maybe needed

### DISCUSSION THREAD

**Discussion by pauby** [chocolatey-community/chocolatey-packages#2450] [General]
It's been [confirmed that changes to the MSI for `cmake` have been made to add it to the PATH by default](https://gitlab.kitware.com/cmake/cmake/-/issues/21465#note_1503498). 

The package for [`cmake`](https://community.chocolatey.org/packages/cmake) and [`cmake.install`](https://community.chocolatey.org/packages/cmake.install) currently states that installer arguments can be used to do this. Depending on what has been done to the MSI, these installer arguments may no longer be valid. In addition, we should update the description with the new changes.

I've added this here to gather others thoughts.

**Comment by AdmiringWorm** [comment]
Do we know which versions this will land in?

If we do, we could update the description to explicitly mention in which version it was changed in.

**Reply by pauby** [reply]
According to the [merge request](https://gitlab.kitware.com/cmake/cmake/-/merge_requests/9386) it is in the 3.30.0 milestone.
